### PR TITLE
Lithuanian translation update

### PR DESCRIPTION
**Pull request by tomass**

### CODE REVIEW

File: core/src/main/resources/jenkins/triggers/ReverseBuildTrigger/config_lt.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Projects\ to\ watch=Steb?ti projektus
```
Comment (daniel-beck) [inline core/src/main/resources/jenkins/triggers/ReverseBuildTrigger/config_lt.properties]: This looks like an encoding issue.

Or Lithuanian is weirder than my imagination can handle 😉

**Comment by tomass** [comment]
Encoding issue has been fixed.

**Comment by oleg-nenashev** [comment]
Retriggering the build

**Comment by tomass** [comment]
Do I understand correctly that there are no compilation problems with translations and it is a problem with jenkins-github integration which makes this PR labelled as "All checks have failed" because build was aborted?

**Comment by oleg-nenashev** [comment]
The build work well now.
Merging the PR

**Pull request merged**